tietokanta/src/main/resources/db/migration/V1_870__.sql

```diff
@@ -0,0 +1,69 @@
+-- Vain osa toimenpidekooditaulun tehtävistä otetaan VEMTR- ja tehtämääräraporteilla mukaan
+-- Jotta tämä saadaan mahdollistettua, meidän pitää lisätä toimenpidekoodi tauluun merkintä,
+-- jossa yksiselitteisesti määritellään nämä tehtävät.
+-- Mukaanotettavat tehtävät on määritelty erillisessä excelissä.
+
+ALTER TABLE toimenpidekoodi
+    ADD "raportoi-tehtava?" BOOLEAN DEFAULT FALSE NOT NULL;
+
+-- Päivitetään raportoi-tehtava? = true, niiltä tehtäviltä, jotka halutaan noihin raportteihin
+-- Joudumme päivittämään nämä nimen perusteella, koska id ei täsmää eri ympäristöissä
+UPDATE toimenpidekoodi
+SET "raportoi-tehtava?" = TRUE
+WHERE (
+              nimi = 'Vakiokokoisten liikennemerkkien uusiminen,  pelkkä merkki'
+              OR nimi = 'Vakiokokoisten liikennemerkkien uusiminen ja lisääminen merkki tukirakenteineen (60 mm varsi)'
+              OR nimi = 'Vakiokokoisten liikennemerkkien uusiminen ja lisääminen merkki tukirakenteineen (90 mm varsi)'
+              OR nimi = 'Opastustaulun/-viitan uusiminen'
+              OR nimi =
+                 'Opastustaulujen ja liikennemerkkien rakentaminen tukirakenteineen (sis. liikennemerkkien poistamisia)'
+              OR nimi = 'Opastustaulujen ja opastusviittojen uusiminen portaaliin'
+              OR nimi = 'Töherrysten poisto'
+              OR nimi = 'Töherrysten estokäsittely'
+              OR nimi = 'Yksityisten rumpujen korjaus ja uusiminen  Ø ≤ 400 mm, päällystetyt tiet'
+              OR nimi = 'Yksityisten rumpujen korjaus ja uusiminen  Ø > 400 mm ≤ 600 mm, päällystetyt tiet'
+              OR nimi = 'Yksityisten rumpujen korjaus ja uusiminen  Ø ≤ 400 mm, soratiet'
+              OR nimi = 'Yksityisten rumpujen korjaus ja uusiminen  Ø > 400 mm ≤ 600 mm, soratiet'
+              OR nimi = 'Kannukaatosaumaus'
+              OR nimi = 'KT-valuasfalttisaumaus'
+              OR nimi = 'Päällystettyjen teiden palteiden poisto'
+              OR nimi = 'Reunapalteen poisto kaiteen alta'
+              OR nimi = 'Maakivien (>1m3) poisto'
+              OR nimi = 'Avo-ojitus/päällystetyt tiet'
+              OR nimi = 'Avo-ojitus/soratiet'
+              OR nimi = 'Avo-ojitus/soratiet (kaapeli kaivualueella)'
+              OR nimi = 'Kalliokynsien louhinta ojituksen yhteydessä'
+              OR nimi = 'Soratien runkokelirikkokorjaukset'
+          );
+
+-- Näitä löytyy samalla nimellä useampia, niin tarvitaan tarkemat hakuehdot
+UPDATE toimenpidekoodi
+SET "raportoi-tehtava?" = TRUE
+WHERE (
```

Comment:
Itse kirjottelisin nämä varmasti muotoon WHERE nimi IN (a, b, c, d). Syntaksi hieman lyhenee.

Reply:
Hyvä pointti. Muutin.